R/TaskClassif_melanoma.R

```diff
@@ -83,7 +83,7 @@ load_task_melanoma = function(id = "melanoma") {
   cached_constructor = function(backend) {
     data = cached(constructor_melanoma, "datasets", "melanoma")$data
 
-    data[, benign_malignant := factor(benign_malignant, levels = c("benign", "malignant"))]
+    data[, outcome := factor(outcome, levels = c("benign", "malignant"))]
```

Comment:
the static analyzer will not know that outcome is a valid variable here (this is generally an issue with NSE)
you can use `outcome := factor(get("outcome"), ...)` as a workaround

Reply:
With this change, the linter gives the following warning: `no visible binding for global variable 'outcome'`. Using `set()` does also. Is this an issue? If I understand correctly, that means the problem is still there


![Screenshot 2024-12-02 at 15 05 11](https://github.com/user-attachments/assets/6d166dd4-43af-45e2-a904-8b032aa111a5)

Reply:
this is also not what I wrote. I said `get("outcome")`